modules/admin-ui-frontend/app/scripts/shared/partials/modals/series-details.html

```diff
@@ -388,39 +388,43 @@ <h2 translate="EVENTS.SERIES.DETAILS.HEADER" translate-values="{{titleParams}}">
       <div class="full-col">
         <div class="obj list-obj">
           <header translate="EVENTS.SERIES.DETAILS.TABS.TOBIRA"></header>
-          <div class="obj-container">
-            <a translate="EVENTS.SERIES.DETAILS.TOBIRA.DIRECT_LINK" href="{{tobiraData.baseURL}}/!s/:{{resourceId}}"></a>
-          </div>
-          <div class="obj-container">
-            <div class="obj tbl-list">
-              <header translate="EVENTS.SERIES.DETAILS.TOBIRA.PAGES"></header>
-              <div class="obj-container">
-                <table class="main-tbl">
-                  <tbody>
-                    <tr ng-if="!tobiraData.hostPages || tobiraData.hostPages.length === 0">
-                      <td class="tobira-not-mounted" translate="EVENTS.SERIES.DETAILS.TOBIRA.NOT_MOUNTED"></td>
-                    </tr>
-                    <tr ng-repeat="hostPage in tobiraData.hostPages">
-                      <td>
-                        <a href="{{tobiraData.baseURL}}{{hostPage.path}}" ng-switch="hostPage.path === '/'">
-                          <div ng-switch-when="false">
-                            <span class="tobira-page-separator">/</span>
-                            <span ng-repeat="ancestor in hostPage.ancestors">
-                              {{ancestor.title}}
+          <div ng-if="!tobiraData.error">
+            <div class="obj-container">
+              <a translate="EVENTS.SERIES.DETAILS.TOBIRA.DIRECT_LINK" href="{{directTobiraLink}}"></a> <a ng-click="copyTobiraDirectLink()" aria-label="{{'EVENTS.SERIES.DETAILS.TOBIRA.COPY_DIRECT_LINK' | translate}}">
+                <i aria-hidden="true" class="fa fa-copy" title="{{'EVENTS.SERIES.DETAILS.TOBIRA.COPY_DIRECT_LINK' | translate}}"></i>
```

Comment:
One additional note from testing this: I would probably style this "copy to clipboard" thing as a button and put it slightly farther away from the link. I think right now, at first glance, everyone would assume the whole thing is one link. Thus I expect some confusion.

Reply:
Fair.